com.unity.cinemachine/Runtime/Components/CinemachineFramingTransposer.cs

```diff
@@ -30,7 +30,7 @@ namespace Cinemachine
     [DocumentationSorting(DocumentationSortingAttribute.Level.UserRef)]
     [AddComponentMenu("")] // Don't display in add component menu
     [SaveDuringPlay]
-    [CameraPipelineAttribute(CinemachineCore.Stage.Body)]
+    [CameraPipeline(CinemachineCore.Stage.Body)]
     public class CinemachineFramingTransposer : CinemachineComponentBase
```

Comment:
This class should also implement Modifiable Screen Position

Reply:
What happens when Composer and FT is on Freelook?
Also, currently FreelookModifier says: No applicable CM components found. Must have one of: Cinemachine3rdPersonFollow, OrbitalFollow, POV. 
Should we add FT and Composer to that list?